File: dotnet/stack/Hosts/Console/AppConfig.cs
using Agience.SDK;

namespace Agience.Hosts._Console
{
    internal class AppConfig : HostConfig
    {
        public string? OpenAiApiKey { get; set; }
        public string? OpenAIApiUrl { get; set; }
        public string? CustomNtpHost { get; set; }
    }
}

File: python/src/hosts/console/app_config.py
from typing import Optional

from core.host_config import HostConfig


class AppConfig(HostConfig):
    openai_api_key: Optional[str] = None
    openai_api_url: Optional[str] = None
    custom_ntp_host: Optional[str] = None
    workspace_path: Optional[str] = None
